docs/notebooks/tutorial.ipynb

```diff
@@ -5274,14 +2772,279 @@
+       "2003-01-03 21:00:00+00:00            0             0 -11.224972   \n",
+       "2003-01-06 21:00:00+00:00            0             0 -12.961481   \n",
+       "2003-01-07 21:00:00+00:00            0             0 -12.961481   \n",
+       "2003-01-08 21:00:00+00:00            0             0  -9.653623   \n",
+       "\n",
+       "                           starting_cash  starting_exposure  starting_value  \\\n",
+       "2003-01-02 21:00:00+00:00     10000000.0                0.0             0.0   \n",
+       "2003-01-03 21:00:00+00:00     10000000.0                0.0             0.0   \n",
+       "2003-01-06 21:00:00+00:00      9999850.0              149.0           149.0   \n",
+       "2003-01-07 21:00:00+00:00      9999700.0              298.0           298.0   \n",
+       "2003-01-08 21:00:00+00:00      9999550.5              445.5           445.5   \n",
+       "\n",
+       "                           trading_days  \\\n",
+       "2003-01-02 21:00:00+00:00             1   \n",
+       "2003-01-03 21:00:00+00:00             2   \n",
+       "2003-01-06 21:00:00+00:00             3   \n",
+       "2003-01-07 21:00:00+00:00             4   \n",
+       "2003-01-08 21:00:00+00:00             5   \n",
+       "\n",
+       "                                                                transactions  \\\n",
+       "2003-01-02 21:00:00+00:00                                                 []   \n",
+       "2003-01-03 21:00:00+00:00  [{'dt': 2003-01-03 21:00:00+00:00, 'amount': 1...   \n",
+       "2003-01-06 21:00:00+00:00  [{'dt': 2003-01-06 21:00:00+00:00, 'amount': 1...   \n",
+       "2003-01-07 21:00:00+00:00  [{'dt': 2003-01-07 21:00:00+00:00, 'amount': 1...   \n",
+       "2003-01-08 21:00:00+00:00  [{'dt': 2003-01-08 21:00:00+00:00, 'amount': 1...   \n",
+       "\n",
+       "                           treasury_period_return  \n",
+       "2003-01-02 21:00:00+00:00                  0.0407  \n",
+       "2003-01-03 21:00:00+00:00                  0.0405  \n",
+       "2003-01-06 21:00:00+00:00                  0.0409  \n",
+       "2003-01-07 21:00:00+00:00                  0.0404  \n",
+       "2003-01-08 21:00:00+00:00                  0.0400  \n",
+       "\n",
+       "[5 rows x 39 columns]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pd.read_pickle('perf_ipython.pickle').head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
    "source": [
-    "If you want to use your own custom bundle, that doesn't use yahoo finance data, check out our [docs](http://www.zipline.io/bundles.html#writing-a-new-bundle) on writing a new bundle"
+    "If you want to use your own custom bundle, check out our [docs](http://www.zipline.io/bundles.html#writing-a-new-bundle) on writing a new bundle"
```

Comment:
Is it clear why one would want to write a custom bundle here? Before it implied that you'd write one to replace the yahoo finance data.